README.md

```diff
@@ -1,10 +1,15 @@
-# 🚀Git Workshop by Basti Ortiz
-
-This repository serves as a check-in for attendees of the **Git Workshop for Modern Software Engineers** hosted by Basti Ortiz, the Director for Engineering of UP CSI.
-
----
-## Attendees
-
----
-## Partners
-This workshop is co-hosted by [Secuna](https://secuna.io), in partnership with UP CURSOR and UP ACM.
+# git-workshop-2023
+
+hi
```

Comment:
but y tho?

Reply:
If ya don't mind, can you also resolve the merge conflicts by syncing your fork's `main` branch, and then merging `main` _into_ `hi-basti`? Let me know if you need help with that.

Reply:
Sure, lemme go ahead with that.